packages/leap/lib/src/leap_game.dart

```diff
@@ -42,6 +42,7 @@ class LeapGame extends FlameGame with HasTrackedComponents {
     required String tiledMapPath,
     required int tileCameraWidth,
     required int tileCameraHeight,
+    String prefix = '',
```

Comment:
This introduces a bug and breaks at runtime. The default value needs to match `TiledComponent.load`:

```
  String prefix = 'assets/tiles/',
```